### ISSUE TITLE
IMG containers kept being in use even after disconnecting from server

### ISSUE THREAD

**Issue by PlatinMTA** [multitheftauto/mtasa-blue#3115]
### Describe the bug

After loading an IMG Container you will not be able to delete or edit it in the same session. You need to restart your MTA for the file to stop being 'in use'.

![image](https://github.com/multitheftauto/mtasa-blue/assets/23291573/6e87e3ac-62e1-4d03-a090-bf7681f89d4d)
_In this picture I'm unable to delete the .IMG file even tho I'm not connected to any server._

### Steps to reproduce

1. Load an IMG Container, for example:
```lua
local imgSkins = engineLoadIMG("skins.img")
engineAddImage(imgSkins)
```

2. After being loaded, you will be unable to change it until you restart your game. Try deleting it.

### Version

Client: 1.6 r21855

### Additional context

This will keep happening even after destroying the IMG Container using ``destroyElement``. This is particulary problematic because if you make changes on your .IMG file via a server update, and the client tries to download the file, it will be downloaded but the file will never be replaced, meaning that the old .IMG file is still on the client. This could let the users use custom skins via your .IMG file by using a custom server.

This shouldn't be a problem tho if you use '@' on your .IMG path, but you shouldn't need to do that in the first place.

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by tederis** [comment]
Should be fixed in r21888(in relation to PR #3110)

**Comment by PlatinMTA** [comment]
Yes, this particular issue is fixed, thank you.

However you can still manipulate MTA to not replace your IMG container (even in admin), which I guess could be done for every file if the client really wanted to.
